x-pack/plugins/infra/server/plugin.ts

```diff
@@ -118,6 +121,20 @@ export class InfraServerPlugin implements Plugin<InfraPluginSetup> {
       metrics: new InfraMetricsDomain(new KibanaMetricsAdapter(framework)),
     };
 
+    const logsAlertClient = initializeAlertClient({
+      name: 'logs',
+      getStartServices: core.getStartServices,
+      plugins,
+      logger: this.logger,
```

Comment:
If this initialize helper were in the ruleRegistry plugin, it could just use its own logger.